tests/catch/unit/module/hipModuleGetFunction.cc

```diff
@@ -44,18 +44,24 @@ TEST_CASE("Unit_hipModuleGetFunction_Negative_Parameters") {
                     hipErrorInvalidValue);
   }
 
+// Disabled on AMD due to defect
+#if HT_NVIDIA
```

Comment:
Is there a ticket for this?

Reply:
There is. I've updated the PR with the ticket IDs added in the comments.